omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -1394,10 +1394,12 @@
                     jq().toastmessage('showSuccessToast', "Client Added to a Refill Group Successfully");
                 },
                 error: function (response) {
-                    jq().toastmessage('showErrorToast', "Error while Added Client to Refill Group");
+                    if (cohort !== ""){
```

Comment:
I think its better when you don't just suppress the error but block off the whole ajax rest action

Reply:
@slubwama I am not suppressing the error, and the ajax call is not called once we have not enabled CDDP categorization

Reply:
this is to ensure that the error is thrown only when we have cohort data captured, because its possible to enable the CDDP categorization but the fields are not filled